### ISSUE TITLE
alertmanager maintenance messages should be DEBUG

### ISSUE THREAD

**Issue by tzz** [prometheus/alertmanager#1346]
**What did you do?**

Look at alertmanager logs at default (INFO) verbosity.

**What did you expect to see?**

Informational messages.

**What did you see instead? Under which circumstances?**

The "maintenance" messages for nflog and silences are showing up at INFO level. They seem to be useful as a debugging tool but not to users, so I propose moving them to DEBUG level.

**Environment**

* Alertmanager version:

0.15 rc1

* Logs:

Every 15 minutes these 4 messages show up:

```
nflog Running maintenance...
silences Running maintenance...
silences Maintenance done.
nflog Maintenance done.

```

**Comment by tzz** [comment]
@SuperQ I can't paste the exact log messages at this time but I hope this is enough to explain the request. Thanks for looking.

**Comment by mxinden** [comment]
Moving these to `debug` sounds reasonable to me. @tzz would you mind proposing a PR for this? Let me know if you need any help.

**Comment by tzz** [comment]
@mxinden done in #1347, thank you!

**Comment by stuartnelson3** [comment]
closed by #1347
